### DISCUSSION TITLE
How to Add a Polyfill for the Array at() Method in Nuxt 3?

### DISCUSSION THREAD

**Discussion by sync42johnny** [nuxt/nuxt#24341] [Questions]
How can I add a polyfill for the at() method in my Nuxt 3 project to achieve cross-browser compatibility?

**Comment by TheAlexLichter** [comment]
You could probably use something like https://polyfill.io/v3/polyfill.min.js?features=Array.prototype.at or get the polyfill "manually". Then, you could load it as script via `useHead` or `nuxt.config.ts`

Alternatively, you can also try setting the build target as described in https://github.com/nuxt/nuxt/issues/19328

**Reply by sync42johnny** [reply]
babel?

**Reply by TheAlexLichter** [reply]
Nope - but I linked another issue above (https://github.com/nuxt/nuxt/issues/19328) that might help by setting the build target 👍
